adafruit_mpu6050.py

```diff
@@ -347,3 +372,17 @@ def cycle_rate(self, value: int) -> None:
             raise ValueError("cycle_rate must be a Rate")
         self._cycle_rate = value
         sleep(0.01)
+
+    @property
+    def clock_source(self) -> int:
+        """Returns current clock source selection"""
```

Comment:
This sounds a little picky, but you change this to say something like "The clock source for the device"?  The reason is that this method (the "getter" method) is what populates in our automatic documentation generation, and right now it will sound like this is a "getter" only property, when in fact you have the setter functionality as well!  You could also say "Returns or sets the the current clock source", up to you, just something that describes either what the property is, or mentions that you can get AND set.

For the same reason, the setter's docstring isn't used in the documentation, so we can keep it for source code but it isn't used by `sphinx`.

Reply:
Thank you, hopefully the new wording js more appropriate.